sites/kit.svelte.dev/src/routes/Banner.svelte

```diff
@@ -16,7 +16,7 @@
 <header class="banner" class:visible>
 	<p>
 		<a href="https://www.youtube.com/watch?v=N4BRVkQVoMc">
-			<strong>Dec 14:</strong> a Svelte Radio live special 👀
+			<strong>LIVE NOW:</strong> announcing SvelteKit 1.0
```

Comment:
this deserves an exclamation mark :grin: 
```suggestion
			<strong>LIVE NOW:</strong> announcing SvelteKit 1.0!
```